pkgs/by-name/libresoc/package.nix

```
@@ -0,0 +1,71 @@
+{
+  callPackage,
+  fetchgit,
+}: let
+  fetchFromLibresoc = {
+    pname,
+    hash,
+    rev,
+    ...
```

```suggestion
    rev
```

Ellipsis is not useful, because only the arguments `url`, `rev`, `hash` are used in the body, and it actually invites callees to pass arguments that are then just not used.

---

Yup, that's good :)